src/components/data/data-utils.js

```diff
@@ -659,14 +673,15 @@ const AttributeInput = ({ value: attributeValue, onChange, entityTypes = [] }) =
         h(Link, {
           style: { display: 'block', marginTop: '1rem' },
           onClick: () => {
+            focusLastListItemInput.current = true
             const newAttributeValue = _.update('items', Utils.append(defaultValue), attributeValue)
             onChange(newAttributeValue)
           }
         }, [icon('plus', { style: { marginRight: '0.5rem' } }), 'Add item'])
       ]) : div({ style: { marginTop: '1.5rem' } }, [
         renderInput({
           'aria-label': 'New value',
-          autoFocus: true,
+          autoFocus,
```

Comment:
With autoFocus set, the attribute value input would take focus as soon as one letter was typed in the attribute name input.